web/packages/teleport/src/Integrations/Enroll/AwsConsole/Access/Access.tsx

```diff
@@ -0,0 +1,255 @@
+    // clear filters on syncAll
+    if (syncAll) {
+      setFilters([]);
+    }
+  }, [syncAll]);
+
+  if (!canEnroll) {
+    return (
+      <FeatureBox>
+        <Alert kind="info" mt={4}>
+          You do not have permission to enroll integrations. Missing role
+          permissions: <code>integrations.create</code>
+        </Alert>
+        <EmptyState />
+      </FeatureBox>
+    );
+  }
+
+  if (!integrationName || !trustAnchorArn || !syncRoleArn || !syncProfileArn) {
+    return (
+      <FeatureBox>
+        <Alert kind="info" mt={4}>
+          Missing form data, please go back and restart enrollment.
+        </Alert>
+        <ButtonPrimary
+          onClick={() =>
+            history.push(
+              cfg.getIntegrationEnrollRoute(
+                IntegrationKind.AWSRa,
+                'integration'
+              )
+            )
+          }
+          width="100px"
+        >
+          Back
+        </ButtonPrimary>
+      </FeatureBox>
+    );
+  }
+
+  return (
+    <>
+      <Flex mt={3} justifyContent="space-between" alignItems="center">
+        <H2>Configure Access</H2>
+        <InfoGuideButton
+          config={{
+            guide: <Guide resourcesRoute={resourceRoute} />,
+          }}
+        />
+      </Flex>
+      <P2 mb={3}>
+        Import and synchronize AWS IAM Roles Anywhere Profiles into Teleport.
+        Imported Profiles will be available as Resources with each Role
+        available as an account.
+      </P2>
+      {status === 'error' && (
+        <Alerts.Danger details={error.message}>
+          Error: {error.name}
+        </Alerts.Danger>
+      )}
+      {status === 'pending' && (
+        <Box data-testid="loading" textAlign="center" m={10}>
+          <Indicator />
+        </Box>
+      )}
+      {status === 'success' &&
+        (!data.profiles || data.profiles.length === 0) && (
+          <ProfilesEmptyState />
+        )}
+      {status === 'success' && data.profiles.length > 0 && (
+        <Profiles
+          data={data.profiles}
+          fetchStatus={isFetching ? 'loading' : ''}
+          filters={filters}
+          setFilters={setFilters}
+          refetch={refetch}
+          syncAll={syncAll}
+          setSyncAll={setSyncAll}
+        />
+      )}
+      {submitSync.error && (
+        <Alerts.Danger details={submitSync.error?.message} mt={2}>
+          Error: {submitSync.error.name}
+        </Alerts.Danger>
+      )}
+      <Flex gap={3} mt={3}>
+        <ButtonPrimary width="200px" onClick={() => submitSync.mutate()}>
+          Enable Sync
+        </ButtonPrimary>
+        <ButtonSecondary
+          onClick={() =>
+            history.push(
+              cfg.getIntegrationEnrollRoute(
+                IntegrationKind.AWSRa,
+                'integration'
+              )
+            )
+          }
+          width="100px"
+        >
+          Back
+        </ButtonSecondary>
+      </Flex>
+    </>
+  );
+}
+
+function ProfilesEmptyState() {
```

Comment:
if we show this state, i think we should disable the `enable sync` at the bottom

<img width="1038" height="546" alt="image" src="https://github.com/user-attachments/assets/b01495a4-eafa-4e7c-ae1c-603de78fa30e" />